docs/source/buildscripts/sdc_build_doc.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# *****************************************************************************
+# Copyright (c) 2019, Intel Corporation All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+#     Redistributions of source code must retain the above copyright notice,
+#     this list of conditions and the following disclaimer.
+#
+#     Redistributions in binary form must reproduce the above copyright notice,
+#     this list of conditions and the following disclaimer in the documentation
+#     and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+# OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+# OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+# EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *****************************************************************************
+
+import os
+import sys
+from setuptools import Command
+
+
+# Sphinx User's Documentation Build
+
+class SDCBuildDoc(Command):
+    description = 'Builds Intel SDC documentation'
+    user_options = [
+        ('format', 'f', 'Documentation output format (html, pdf)'),
+        ('doctype', 't', 'Documentation type (user, dev)'),
+    ]
+
+    @staticmethod
+    def _remove_cwd_from_syspath():
+        # Remove current working directory from PYTHONPATH to avoid importing SDC from sources vs.
+        # from installed SDC package
+        cwd = os.getcwd()
+
+        removed_flag = True
+        while removed_flag:
+            removed_flag = False
+            if '' in sys.path:
+                sys.path.remove('')
+                removed_flag = True
+
+            if '.' in sys.path:
+                sys.path.remove('.')
+                removed_flag = True
+
+            if './' in sys.path:
+                sys.path.remove('./')
+                removed_flag = True
+
+            if cwd in sys.path:
+                sys.path.remove(cwd)
+                removed_flag = True
```

Comment:
You could do that in the loop:
```
for pth in ['', '.', './'. cwd]:
    sys.path.remove(pth)
    removed_flag = True
```

Reply:
Correct replacement:
```
for pth in ['', '.', './'. cwd]:
    if pth in sys.path:
        sys.path.remove(pth)
        removed_flag = True
```
`sys.path.remove(x)` raises `ValueError: list.remove(x): x not in list`